src/cxx_support.rs

```diff
@@ -0,0 +1,214 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Support for cxx types.
+
+use std::pin::Pin;
+
+use cxx::{memory::UniquePtrTarget, UniquePtr};
```

Comment:
Nit: I prefer to use single-item-per-import for normal imports. Would you mind unrolling this into two use items?

Reply:
Done